src/Sarif.FunctionalTests/SarifConverterTestData/AndroidStudio/packageEmpty.xml.sarif

```diff
@@ -27,13 +35,7 @@
                   "startLine": 242
                 }
               },
-              "logicalLocation": [
-                {
-                  "name": "northwindtraders-droid-2",
-                  "kind": "android-module"
-                }
-              ],
-              "fullyQualifiedLogicalName": "northwindtraders-droid-2"
+              "logicalLocation": "northwindtraders-droid-2"
```

Comment:
logical location of an issue that resolves to Model.Java consists of only the module? Seems wrong.